src/main.ts

```diff
@@ -17,11 +17,12 @@ interface Cursor<T> {
   userMetaData: UserMetadata<T>
 }
 
-export function defaultCursorRenderer<T>(cursor: Cursor<T>): HTMLElement {
-  const htmlFragment = `<div id="cursor_${cursor.id}" class="cursor">
-    <svg
+function getSvgForCursor(color: string) {
+  return `<svg
       xmlns="http://www.w3.org/2000/svg"
-      viewBox="0 0 35 35"
+      viewBox="13 9 35 35"
```

Comment:
Wait why are we changing the viewbox

Reply:
basically it is off-center at 0,0 so i manually adjusted it in an svg viewer so that it is centered. You don't notice this when its other people's cursors but you do for your own cursor. I think this also accounts for some of the offness when we were trying to sync up fingerprints bc the cursor was getting rendered off to the side from where you were actually clicking